src/main/java/Chappy.java

```diff
@@ -0,0 +1,244 @@
+
+                        case DEADLINE:
+                            String[] deadlineInput = userInput.trim().split("(?i)" + Command.DEADLINE.getKeyword());
+                            if (deadlineInput.length < 2) {
+                                throw new CreateTaskException("Oh SIR! The description of a Deadline cannot be empty!");
+                            }
+                            Deadline.validateOptions(userInput.trim());
+                            String[] deadlineInput2 = deadlineInput[1].split(Deadline.Option.values()[0].getKeyword());
+                            Deadline deadline = new Deadline(deadlineInput2[0].trim(), deadlineInput2[1].trim());
+                            Task.addTask(deadline);
+                            Chappy.saveToDisk();
+                            break;
+
+                        case EVENT:
+                            String[] eventInput = userInput.trim().split("(?i)" + Command.EVENT.getKeyword());
+                            if (eventInput.length < 2) {
+                                throw new CreateTaskException("Oh SIR! The description of an Event cannot be empty!");
+                            }
+                            Event.validateOptions(userInput.trim());
+                            String[] eventInput2 = eventInput[1].split(Event.Option.values()[0].getKeyword());
+                            String[] eventInput3 = eventInput2[1].split(Event.Option.values()[1].getKeyword());
+                            Event task = new Event(eventInput2[0].trim(), eventInput3[0].trim(), eventInput3[1].trim());
+                            Task.addTask(task);
+                            Chappy.saveToDisk();
+                            break;
+
+                        case DELETE:
+                            String[] deleteInput = userInput.trim().split("(?i)" + Command.DELETE.getKeyword());
+
+                            if (deleteInput.length < 2) {
+                                throw new CreateTaskException(
+                                        "Oh SIR! The index input of a Delete command cannot be empty!");
+                            }
+                            int deleteIndex = Integer.parseInt(deleteInput[1].trim()) - 1;
+                            if (deleteIndex < 0 || deleteIndex > userInputArray.size() - 1) {
+                                throw new CreateTaskException("Oh SIR! That task index does not exist!");
+                            }
+                            Task.removeTask(deleteIndex);
+                            Chappy.saveToDisk();
+                            break;
+                    }
+                } else {
+                    throw new CreateTaskException("Oh SIR! I can't understand what you are saying!");
+                }
+
+            } catch (CreateTaskException e) {
+                System.out.println(e.getMessage());
+
+            }
+
+        }
+    }
+
+    public static void saveToDisk() throws IOException {
```

Comment:
Who knows in 20 years those kids won't know whats a disk 🤣 
Perhaps can consider local storage if you like?